horace_core/sqw/@sqw/tobyfit_DGdisk_resconv.m

```diff
@@ -166,132 +243,78 @@
     mod_shape_mono_table.object_store = mod_shape_mono;
 end
 
-reset_state=caller.reset_state;
-for i=1:numel(ind)
-    % Get index of workspace into lookup tables
-    iw=ind(i);
 
-    % Set random number generator if necessary, and save if required for later
-    if reset_state
-        if ~isempty(state_in{i})
-            rng(state_in{i})
-        end
-    else
-        state_out{i} = rng;     % capture the random number generator state
-    end
+% Recompute Q on-the-fly
+qw = calculate_q(ki(irun), kf, detdcn, spec_to_rlu(:,:,irun));
```

Comment:
The cell array 'qw' no longer contains energy (which is the origin of the w standing in for an omega) - rename to, say, just 'q' to avoid confusion.